src/Examples/Advanced/CustomProperties/Hydrocarbon_Processing_example.ipynb

```diff
@@ -151,6 +154,28 @@
+      "2021-12-21 14:02:09 [WARNING] idaes.generic_models.properties.core.pure.Perrys: DEPRECATED - fs.VLE_props.heptene dens_mol_liq_comp_coeff index 'eqn_type' should be specified, defaulting to equation form 1.\n",
+      "2021-12-21 14:02:09 [WARNING] idaes.generic_models.properties.core.pure.Perrys: DEPRECATED - fs.VLE_props.octene dens_mol_liq_comp_coeff index 'eqn_type' should be specified, defaulting to equation form 1.\n",
+      "2021-12-21 14:02:09 [WARNING] idaes.generic_models.properties.core.pure.Perrys: DEPRECATED - fs.Vap_props.ethane dens_mol_liq_comp_coeff index 'eqn_type' should be specified, defaulting to equation form 1.\n",
+      "2021-12-21 14:02:09 [WARNING] idaes.generic_models.properties.core.pure.Perrys: DEPRECATED - fs.Vap_props.propane dens_mol_liq_comp_coeff index 'eqn_type' should be specified, defaulting to equation form 1.\n",
+      "2021-12-21 14:02:09 [WARNING] idaes.generic_models.properties.core.pure.Perrys: DEPRECATED - fs.Vap_props.nbutane dens_mol_liq_comp_coeff index 'eqn_type' should be specified, defaulting to equation form 1.\n",
+      "2021-12-21 14:02:09 [WARNING] idaes.generic_models.properties.core.pure.Perrys: DEPRECATED - fs.Vap_props.ibutane dens_mol_liq_comp_coeff index 'eqn_type' should be specified, defaulting to equation form 1.\n",
+      "2021-12-21 14:02:09 [WARNING] idaes.generic_models.properties.core.pure.Perrys: DEPRECATED - fs.Vap_props.ethylene dens_mol_liq_comp_coeff index 'eqn_type' should be specified, defaulting to equation form 1.\n",
+      "2021-12-21 14:02:09 [WARNING] idaes.generic_models.properties.core.pure.Perrys: DEPRECATED - fs.Vap_props.propene dens_mol_liq_comp_coeff index 'eqn_type' should be specified, defaulting to equation form 1.\n",
+      "2021-12-21 14:02:09 [WARNING] idaes.generic_models.properties.core.pure.Perrys: DEPRECATED - fs.Vap_props.butene dens_mol_liq_comp_coeff index 'eqn_type' should be specified, defaulting to equation form 1.\n",
+      "2021-12-21 14:02:09 [WARNING] idaes.generic_models.properties.core.pure.Perrys: DEPRECATED - fs.Vap_props.pentene dens_mol_liq_comp_coeff index 'eqn_type' should be specified, defaulting to equation form 1.\n",
+      "2021-12-21 14:02:10 [WARNING] idaes.generic_models.properties.core.pure.Perrys: DEPRECATED - fs.Vap_props.hexene dens_mol_liq_comp_coeff index 'eqn_type' should be specified, defaulting to equation form 1.\n",
+      "2021-12-21 14:02:10 [WARNING] idaes.generic_models.properties.core.pure.Perrys: DEPRECATED - fs.Vap_props.heptene dens_mol_liq_comp_coeff index 'eqn_type' should be specified, defaulting to equation form 1.\n",
+      "2021-12-21 14:02:10 [WARNING] idaes.generic_models.properties.core.pure.Perrys: DEPRECATED - fs.Vap_props.octene dens_mol_liq_comp_coeff index 'eqn_type' should be specified, defaulting to equation form 1.\n",
```

Comment:
I noticed there are some warnings in the output (presumably from the new code added to reduce the runtime?). Personally I'm not bothered by them (especially since from my point of view, the alternative to a notebook with warnings is a notebook that cannot be run), so I'm just bringing this up as something to keep in mind.

Reply:
These warnings are not from my changes and are expected since the thermodynamic properties do not specify an equation form for one of the VLE calculated properties. @andrewlee94 and I added this as part of https://github.com/IDAES/idaes-pse/pull/543, and the warnings mean that it is working as expected.

Reply:
I see, thanks for the clarification on both sets of warnings!